### PR TITLE
[Cortex XDR] support for multiple endpoint statuses

### PR DESCRIPTION
**Pull request by ennozdd**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Added support for multiple endpoint statuses for the command *xdr-get-endpoints-by-status*

## Screenshots
image

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.yml [hunk 1]
```diff
@@ -766,6 +766,12 @@ script:
       - disconnected
       - lost
       - uninstalled
+      - windows
```
Comment (ennozdd) [inline Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.yml:769]: image

https://cortex-panw.stoplight.io/docs/cortex-xdr/f922236a8bc9a-get-endpoint

**Review by samuelFain — COMMENTED** [review]
@ennozdd Thank you so much for your contribution, I appreciate your patience!
Added a few minor comments - please review them.

Also, since you changed `xdr-get-endpoints-by-status` command `status` argument to support a list of values, it will be appropriate to change the command argument description accordingly in CortexXDRIR.yml
For example: 'A comma-separated list of endpoint statuses to filter.'

File: Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.yml [hunk 2]
```diff
@@ -758,14 +758,20 @@ script:
       - 'true'
       - 'false'
   - arguments:
-    - description: The status of the endpoint to filter.
+    - description: A comma-separated list of endpoint statuses to filter.
       name: status
       auto: PREDEFINED
       predefined:
```
Comment (samuelFain) [inline Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.yml:764]: Just noticed this,
If we want to allow a comma separated list as a command argument input, we should add `isArray: true`.
Also, for the same `status` argument, we have a predefined list of values, which as far as i know allows only one value as an input and not an array.
I think it should be one or the other and not both.
If the intended use is as your screenshot in the PR's description, i think the array option is the way to go here.
(The predefined list of options can be added in the description to help the users).

**Comment by ennozdd** [comment]
@samuelFain done

File: Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.yml [hunk 3]
```diff
@@ -2694,15 +2689,10 @@ script:
       description: The integration name of the endpoint vendor.
       type: String
   - arguments:
-    - auto: PREDEFINED
-      description: The status of the endpoint to filter.
+    - description: The status of the endpoint to filter.
```
Comment (samuelFain) [inline Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.yml:2692]:
```suggestion
    - description: 'A comma-separated list of enpoint statuses to filter. Valid values are: connected, disconnected, lost, uninstalled, windows, linux, macos, android, isolated, unisolated'
```

**Review by samuelFain — COMMENTED** [review]
@ennozdd Amazing job!
Please correct this last description to mach with the new and better description for `status` argument, so that we may proceed to merge this PR.

**Comment by ennozdd** [comment]
@samuelFain done

**Review by samuelFain — APPROVED** [review]

**Pull request merged**